src/commands/admin/count.ts

```diff
@@ -0,0 +1,30 @@
+import { ADMIN_PERMS } from '@lib/permissions';
+import { Command } from '@lib/types/Command';
+import { ApplicationCommandOptionData, ApplicationCommandPermissionData, CategoryChannel, CommandInteraction } from 'discord.js';
+
+export default class extends Command {
+
+	description = 'Count channels in a category';
```

Comment:
For future admins, I would suggest mentioning that this is useful for archiving since we'll know when to switch archive channels in the config.